frontend/src/scenes/instance/AsyncMigrations/asyncMigrationsLogic.ts

```diff
@@ -90,9 +98,9 @@ export const asyncMigrationsLogic = kea<asyncMigrationsLogicType>({
             message,
         }),
         closeAsyncMigrationsModal: true,
-    },
+    }),
 
-    reducers: {
+    reducers({
         activeTab: [AsyncMigrationsTab.Management as AsyncMigrationsTab, { setActiveTab: (_, { tab }) => tab }],
```

Comment:
If we're changing tabbing, we should also be making the page behave properly and follow proper routing patterns.